src/main/java/hudson/plugins/git/GitChangeLogParser.java

```diff
@@ -71,7 +73,7 @@ public GitChangeLogParser(GitClient git, boolean authorOrCommitter) {
     }
     
     public List<GitChangeSet> parse(@NonNull InputStream changelog) throws IOException {
-        try (InputStreamReader isr = new InputStreamReader(changelog, StandardCharsets.UTF_8); LineIterator it = new LineIterator(isr)) {
+        try (Reader r = new InputStreamReader(changelog, StandardCharsets.UTF_8); LineIterator it = new LineIterator(r)) {
```

Comment:
More readable to use the generic interface on the left side and the more specific implementation on the right side.